manifests/globals.pp

```diff
@@ -106,7 +106,11 @@
     '93'    => '2.1',
     default => undef,
   }
-  $globals_postgis_version = pick($postgis_version, $default_postgis_version)
+  if $postgis_version {
```

Comment:
Since we assume later on that the minimal version is 2  we should default to that as a nested pick, but we fail if we don't get it either way.